dev/benchmarks/macrobenchmarks/lib/src/web/bench_card_infinite_scroll.dart

```diff
@@ -0,0 +1,79 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+
+import 'recorder.dart';
+import 'test_data.dart';
+
+/// Creates an infinite list of Material cards and scrolls it.
+class BenchCardInfiniteScroll extends WidgetRecorder {
+  BenchCardInfiniteScroll() : super(name: benchmarkName);
+
+  static const String benchmarkName = 'bench_card_infinite_scroll';
+
+  @override
+  Widget createWidget() {
+    return MaterialApp(
+      title: 'Infinite Card Scroll Benchmark',
+      home: _InfiniteScrollCards(),
+    );
+  }
+}
+
+class _InfiniteScrollCards extends StatefulWidget {
+  _InfiniteScrollCards({Key key}) : super(key: key);
+
+  @override
+  State<_InfiniteScrollCards> createState() => _InfiniteScrollCardsState();
+}
+
+class _InfiniteScrollCardsState extends State<_InfiniteScrollCards> {
+  ScrollController scrollController;
+
+  double offset;
+  static const double distance = 500;
```

Comment:
We use a 1024 x 1024 window, which would fit 11 cards (based on the 100 height set below). Maybe scroll 1000-2000 pixels so it scrolls at least one full page worth of cards?

Reply:
Done.